app/react/docker/containers/CreateView/InnerForm.tsx

```diff
@@ -51,7 +51,7 @@ export function InnerForm({
   const environment = envQuery.data;
 
   return (
-    <Form className="form-horizontal">
+    <Form className="form-horizontal" autoComplete="off">
```

Comment:
Should this be for the whole form? Are there places where we might still want autocomplete?

Reply:
this is the pattern we have been using - turning off for the whole form. I can ask James in the standup. Andrew has not complaints so far as it turns off the browser autofill not Portainer autofill.